packages/@react-spectrum/listbox/docs/ListBox.mdx

```diff
@@ -127,6 +128,24 @@ function Example() {
   );
 }
 ```
+
+## Links
+
+By default, interacting with an item in a ListBox triggers `onSelectionChange`. Alternatively, items may be links to another page or website. This can be achieved by passing the `href` prop to the `<Item>` component. Link items in a ListBox are not selectable.
+
+```tsx example
+<ListBox aria-label="Links">
+  <Item href="https://adobe.com/" target="_blank">Adobe</Item>
+  <Item href="https://apple.com/" target="_blank">Apple</Item>
+  <Item href="https://google.com/" target="_blank">Google</Item>
+  <Item href="https://microsoft.com/" target="_blank">Microsoft</Item>
+</ListBox>
+```
```

Comment:
I don't like this example because you probably shouldn't use a listbox if you just have a list of links that doesn't support selection. But the example where the links drive the selection state via a client side router would be much larger...